### DISCUSSION TITLE
November | Community Manager Monthly Report - Jesse(JDubs)

### DISCUSSION THREAD

**Discussion by 2xJDubs** [ringdao-community-guild/discussion#200] [Show and tell]
# Overview

November picked up the pace along with the market.  Darwinia migration also started which really activated the community and it seems the scammers, so I spent a lot of time cleaning out scammers and supporting the community in migrating.  I also worked with Laki to help understand some of the nuances of the new staking method and some of the bugs.  I have been looking at opportunities to reach out to project along with supporting projects we are working with currently. The initiatives worked on in November are as follows:

- Management of the DAO staking on InvArch and Tinkernet
- Staking transition from V1 to V2 for Darwinia
- Supporting the RingDAO Marketing Action plan.
- Discussing ideas with the community on Github

# Details
## Community Engagement

**Weekly News**
Working with June, our DCDAO Ambassador, to collate weekly news that is then posted on the DCDAO Twitter account. 

**Monthly DCDAO Update**
Working with June and David on a monthly medium article summarizing what happened with DCDAO the previous month.

#Community Management

- Quiz Games # 50, # 51, # 52, and # 53 were run on Discord with a typical 20 or so members joining.
- Continued to engage and help our community members with staking and migration.
- Supporting the community with KtonDAO initiatives and ideas.
- Supporting the community as we ramp up RingDAO initiatives.

# December Plan

I expect to be doing the following:

- DCDAO to host Quizzes each week
- Continued engagement with the Community
- Continue Weekly News and Monthly Update
- Work with our Advocates to help the maximize their efforts and determine how best they can support DCDAO
- Continued management of the Ambassador and Advocate programs
- Manage the DAO staking and rewards distribution
- Possibly Work with Avail Cross-community support and marketing
- Work with community members to support KtonDAO initiatives
- Continue helping with the Marketing Initiative
- Reaching out to the Solarflare team to discuss cross chain swaps again
- Working with the Sublayer team to support deposit NFTs and other features that will be helpful
- Supporting our building connections in Cosmos and Cardano

# Funds Wallet

Please remit salary to 0xebd9A48eD1128375EB4383ED4d53478B4FD85a8D

**Comment by hackfisher** [comment]
Thanks for the info.

FYI, https://github.com/orgs/dcdao/discussions/201
